cluster/manifests/wiz/sensor/secrets.yaml

```diff
@@ -0,0 +1,39 @@
+{{ if eq .Cluster.ConfigItems.wiz_enable_runtime_monitoring_daemonset "true"}}
+---
+# Source: wiz-sensor/templates/apikeysecret.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: wiz-sensor-apikey
+  namespace: wiz
+  labels:
+    
+    helm.sh/chart: wiz-sensor-1.0.4760
+    image/tag: v1
+    app.kubernetes.io/name: wiz-sensor
+    app.kubernetes.io/instance: wiz-sensor
+    app.kubernetes.io/version: "1.0.4708"
+    app.kubernetes.io/managed-by: Helm
+type: Opaque
+data:
+  clientId: "{{ .Cluster.ConfigItems.wiz_api_client_id | base64 }}"
+  clientToken: "{{ .Cluster.ConfigItems.wiz_api_client_token | base64 }}"
```

Comment:
I suppose the same applies here, do we need to drop the `base64` template function?